markdown/how.md

```diff
@@ -3,17 +3,87 @@
 
 ## OpenStack
 
+<!-- Note -->
+
+Our cloud infrastructure runs on OpenStack, and we use it for both the infrastructure of the learning platform and our lab environments.
+
+With the native installation, we used Openstack Heat to deploy our learning platforms to OpenStack VMs. After transitioning to Docker-based deployment, i.e., containerizing the services and running them on Kubernetes, we use Opnetsack magnum to deploy our Kubernetes cluster.
```

Comment:
This is the first time you mention "native installation", without explaining what it is. You must explain that "native installation" was Open edX lingo for the installation driven by Ansible roles and playbooks. (It was called "native" because it deployed directly onto cloud VMs or baremetal servers, rather than into Vagrant-managed VMs.)